sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/MultiPathImporter.java

```diff
@@ -40,15 +39,15 @@ public MultiPathImporter(SingleFileReportImporter importer, String pathsKey, Str
   }
 
   @Override
-  public void importReport(SensorContext context, Map<String, Integer> numberOfLinesOfCode) {
+  public void importReport(SensorContext context) {
     final String[] paths = context.settings().getStringArray(pathsKey);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)